ProjectProposal/ProjectDescription.md

```diff
@@ -0,0 +1,30 @@
+# Project Description
+
+## What is it?
+Elasticsearch is an open-source project that functions as a search engine that allows you to store/organize raw data, search for keywords and tags, and analyze this data relatively quickly. Elasticsearch also scales to the larger extremes and is therefore useful in nearly any application or setting that requires storing and retrieving large amounts of information.
+
+The backend componenents of Elasticsearch is made up of nodes, clusters, indexes, document, shards, and replicas. A node represents a single server and is a unit of a cluster, it stores data and has a universal identifier. Clusters allow searching and indexing. 
+
+Documents are the basic units that are indexed in elastic search, and are collected in an index. Indexes can also be divided into multiple shards, each acting as their own index, this also allows for multiple copies of the same document.
```

Comment:
Should "elastic search" be "Elasticsearch"?

Reply:
Yes it should, let me fix that.